src/dev-app/drag-drop/drag-drop-demo.ts

```diff
@@ -74,4 +77,16 @@ export class DragAndDropDemo {
       );
     }
   }
+
+  constrainPosition: CdkDrag['constrainPosition'] = (
```

Comment:
Maybe I'm misunderstanding this, but is the idea of this function that the top/left of the item should align with the cursor. It seems like after the fix that's not the case anymore.

Reply:
I am pretty sure the `constrainPosition` has not changed behavior, and it is still the same as was introduced in #25061. 

In this example, I am just returning the original top/left corner of the element, basically not constraining it to anything / changing the position. So it is just to show that the element behaves well given a constrain that basically tells it to stay in place.

Reply:
@crisbeto  What are your closing thoughts here? Does it make sense to you, that for the element not snap to the cursor and stay in place you need to subtract the pickup position? Is this how you expected the final `constrainPosition` to behave? Before doing these changes we should be 100% sure they are not being done due to a mismatch of what you expected the function to do before #25061.

Reply:
Yeah it makes sense now. I was mostly trying to figure out what it was supposed to do in the demo.